xcodeproj/internal/default_automatic_target_processing_aspect.bzl

```diff
@@ -88,6 +89,11 @@ def _default_automatic_target_processing_aspect_impl(target, ctx):
         infoplists = ("infoplists")
         should_generate_target = False
     elif AppleBundleInfo in target:
+        if "codesignopts" in attrs:
+            codesignopts = "codesignopts"
+        entitlements = "entitlements"
```

Comment:
These three have a bad merge. Can you add the `if` checks here and remove the versions below as part of your reordering?